porter/v1/cluster_control_plane.proto

```diff
@@ -487,6 +487,17 @@ message AppHelmValuesResponse {
   string b64_values = 1;
 }
 
+// ManualJobRunRequest is the request object for ManualJobRun
+message ManualJobRunRequest {
```

Comment:
Talked about making this more general - support `porter app run` here where if the service is a job, it will make a manual run and not delete the job pod. If it is a worker or web service, we will create a copy of the existing service and delete it when disconnected